Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -16,85 +16,145 @@
+            if val.count < 2 || !nameString.lowercased().hasSuffix(val.lowercased()) {
+                return val.capitlizeFirstLetter
+            }
+            return nil
+        }
+        if let genericParams = self.genericTypeParams {
+            let genericTypeNames = genericParams.map { $0.name.capitlizeFirstLetter + $0.type.displayName }
+            args.append(contentsOf: genericTypeNames)
+        }
+            
+        args.append(contentsOf: paramTypes.map(path: \.displayName))
+        var dtype = self.type.displayName
+        let capped = min(dtype.count, 32)
+        dtype.removeLast(dtype.count-capped)
+        args.append(dtype)
+        let ret = args.filter{ arg in !arg.isEmpty }
+        return ret
+    }()
+    
+    
+    lazy var handler: ClosureModel? = {
+        if isInitializer {
+            return nil
+        }
+        
+        let paramNames = self.params.map(path: \.name)
+        let paramTypes = self.params.map(path: \.type)
+        let ret = ClosureModel(name: name,
+                               genericTypeParams: genericTypeParams,
+                               paramNames: paramNames,
+                               paramTypes: paramTypes,
+                               suffix: suffix,
+                               returnType: type,
+                               staticKind: staticKind)
+        
+        return ret
+    }()
+    
+    
+    init(name: String,
+         typeName: String,
+         acl: String?,
+         attributes: [String],
+         genericTypeParams: [ParamModel]?,
+         params: [ParamModel],
+         throwsOrRethrows: String?,
+         isStatic: Bool,
+         isInitializer: Bool,
+         offset: Int64,
+         length: Int64,
+         modelDescription: String?,
+         processed: Bool) {
+        self.name = name.trimmingCharacters(in: .whitespaces)
+        self.type = Type(typeName.trimmingCharacters(in: .whitespaces))
+        self.suffix = throwsOrRethrows ?? ""
+        self.attributes = attributes
+        self.offset = offset
+        self.length = length
+        self.isInitializer = isInitializer
+        self.isStatic = isStatic
+        self.params = params
+        self.genericTypeParams = genericTypeParams
+        self.processed = processed
+        self.modelDescription = modelDescription
+        self.accessControlLevelDescription = acl ?? ""
+    }
+    
     init(_ ast: Structure, filepath: String, data: Data, processed: Bool) {
         // This will split func signature into name and the rest (params, return type). In case it's a generic func,
         // its type parameters will be in its substrctures (and < > are omitted in the func ast.name), so it will only
         // give the name part that we expect.
         var comps = ast.name.components(separatedBy: CharacterSet(arrayLiteral: ":", "(", ")")).filter {!$0.isEmpty}
         let nameString = comps.removeFirst()
         self.filePath = filepath
-
+        
```

Comment:
nit: In Xcode, could you go to `Preferences -> Text Editing -> Editing -> Automatically trim trailing whitespace` so we don't have these lines?

Reply:
it's already set to that